packages/suite/jest.config.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
     ],
     coverageThreshold: {
         global: {
-            statements: 63,
+            statements: 62.7,
```

Comment:
how come? is this necessary? you've removed a lot of code yet coverage dropped?

Reply:
No idea, but I added some tests now, so it does not nedd to be lowered.